_layouts/location.html

```diff
@@ -124,11 +125,11 @@ <h4 class="card-title font-family-condensed-extra font-size-h3 text-uppercase">{
 
 <!-- Happenings -->
 {% if location_happenings.size > 0 %}
-<div class="container">
+<div class="container" id="happenings">
```

Comment:
we will add a `data-smooth-scroll-to="happenings"` to facilitate using the smooth scroller